2006Redone Server/src/redone/game/content/guilds/Guilds.java

```diff
@@ -115,8 +115,8 @@ public static void movePlayer(Client c, int objectId) {
 	public static boolean checkRequirments(Client c, int objectId) {
 		switch (objectId) {
 		case 1805: // Champions Guild
-			if (c.questPoints < QuestAssistant.MAXIMUM_QUESTPOINTS) {
-				c.getDialogueHandler().sendStatement("You need " + QuestAssistant.MAXIMUM_QUESTPOINTS + " quest points to enter this guild!");
+			if (c.questPoints < 19) {

```

Comment:
Make it you need 32 or MAX, that way until we reach 32, anyone with all of them completed can get in